threat_intel/opendns.py

```diff
@@ -265,3 +270,8 @@ def latest_malicious(self, ips):
         api_name = 'opendns-latest_malicious'
         fmt_url_path = u'ips/{0}/latest_domains'
         return self._multi_get(api_name, fmt_url_path, ips)
+
+
+class ResponseError(Exception):
+
+    """Raised when the response received from the endpoint is not valid."""
```

Comment:
Please include something here about why this response happened, like "No response for cached domains"